content/docs/user-guide/managing-external-data.md

```diff
@@ -66,21 +66,23 @@ $ dvc run -d data.txt \
 
 ```dvc
 # Add SSH remote to be used as cache location for SSH files
-$ dvc remote add sshcache ssh://user@example.com:/cache
+$ dvc remote add sshcache ssh://user@example.com/home/.../cache
```

Comment:
`/cache` is absolute anyway. `...` is clearly not needed.

Reply:
`/cache` is an absolute path from a SSH system root? Not getting this.

`...` means "more stuff here", what do you mean it's clearly not needed?

Reply:
p.s. Updating the `...` part per https://github.com/iterative/dvc.org/pull/1649#pullrequestreview-459592083 below anyway.